frontend/packages/console-shared/src/components/popover/Popover.tsx

```diff
@@ -49,7 +49,7 @@ const Popover: React.FC<PopoverProps> = ({
           modifiers: { arrow: { element: '.ocs-popover__arrow' } },
         }}
       >
-        <FocusTrap>
+        <FocusTrap translate="no">
```

Comment:
This is [an existing issue in the code base](https://github.com/search?q=repo%3Aopenshift%2Fconsole+translate%3D%22no%22&type=code), and it's not clear why it exists.